src/modules/facility-investor/dto/get-facility-investors-response.dto.ts

```diff
@@ -0,0 +1,71 @@
+import { ApiProperty, ApiResponseProperty } from '@nestjs/swagger';
+import { EXAMPLES, PROPERTIES } from '@ukef/constants';
+import { UkefId } from '@ukef/helpers';
+import { DateOnlyString } from '@ukef/helpers/date-only-string.type';
+
+export type GetFacilityInvestorsResponse = GetFacilityInvestorResponseItem[];
+
+export class GetFacilityInvestorResponseItem {
+  @ApiResponseProperty({ example: PROPERTIES.GLOBAL.portfolioIdentifier })
+  portfolioIdentifier: string;
```

Comment:
Added description and made `readonly` in code. 
decorator ApiResponseProperty was changed to decorator ApiProperty because responses doesn't support description. This also makes it consistent with decorator used for other fields.